src/lib/components/Toast.svelte

```diff
@@ -20,7 +20,7 @@
 	<svelte:component this={icon} />
 	<div class="text">{@html message}</div>
 	<button class="close shrink" on:click={() => dispatch('close')}>
```

Comment:
Don't forget the vision impaired 🥺
```suggestion
	<button aria-label="Close" class="close shrink" on:click={() => dispatch('close')}>
```